examples/experiment-1.0.amp.html

```diff
@@ -27,11 +27,25 @@ <h1>Title TBD</h1>
           "variants": {
             "yellow": {
               "weight": 50,
-              "mutations": [{}]
+              "mutations": [
+                {
+                  "type": "attributes",
+                  "target": "h1",
+                  "attributeName": "style",
+                  "value": "color: yellow"
```

Comment:
we may need to change this example later, given we are not going to support them in mvp 😢